tests/components/bond/test_init.py

```diff
@@ -61,6 +63,7 @@ async def test_async_setup_entry_sets_up_hub_and_supported_domains(hass: HomeAss
     assert len(mock_cover_async_setup_entry.mock_calls) == 1
     assert len(mock_fan_async_setup_entry.mock_calls) == 1
     assert len(mock_light_async_setup_entry.mock_calls) == 1
+    assert len(mock_switch_async_setup_entry.mock_calls) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5844602)